packages/fluentui/react-northstar/src/components/Card/CardTopControls.tsx

```diff
@@ -0,0 +1,71 @@
+import { Accessibility } from '@fluentui/accessibility';
+import { getElementType, getUnhandledProps, useAccessibility, useStyles, useTelemetry } from '@fluentui/react-bindings';
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import { FluentComponentStaticProps, ProviderContextPrepared, WithAsProp, withSafeTypeForAs } from '../../types';
+import { ChildrenComponentProps, commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+
+export interface CardTopControlsProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * */
+  accessibility?: Accessibility<never>;
+}
+
+export type CardTopControlsStylesProps = never;
+
+const CardTopControls: React.FC<WithAsProp<CardTopControlsProps>> &
```

Comment:
I don't like this component, but I don't have better ideas. Let's stick with for now